libs/metrics-wai/src/Data/Metrics/Middleware.hs

```diff
@@ -22,7 +22,6 @@ import qualified Data.Text              as T
 import qualified Data.Text.Encoding     as T
```

Comment:
Isn't this entire module going to be replaced by the prometheus middleware, as called in spar?  If so (or if not), please comment on this.  (Unless you think it'll be obvious to me from the compiler messages once I start removing collectd support.)

Reply:
Yes it will be made obsolete. What's your recommended way of doing so?

Reply:
not sure.  perhaps mention [here](https://github.com/wireapp/wire-server/blob/b9dfb66186798ea44a9be3b5bd9218f9c66be65b/libs/metrics-core/src/Data/Metrics.hs#L189-L201) that the entire `metrics-wai` package will go away with this, too?